README.md

```diff
@@ -162,9 +162,13 @@ This contains an Apache Airflow provider that providers the `@task.venv` decorat
 
 The `# syntax` line tells buildkit to user our Build frontend to process the Dockerfile into instructions.
 
-The example Dockerfile above gets converted into roughly following instructions
+The example Dockerfile above gets converted into roughly the following Dockerfile:
 
 ```Dockerfile
+# syntax=docker/dockerfile:1
```

Comment:
```suggestion
# syntax=docker/dockerfile:1.4
```

Reply:
Well worked for me with 1 too, maybe it takes the latest 1.x so all good

Reply:
It does yeah, `:1` is a movable tag.